courses/models.py

```diff
@@ -664,6 +664,19 @@ class Meta:
     def get_certified_object_id(self):
         return self.course_run_id
 
+    @property
+    def link(self):
+        """
+        Get the link at which this certificate will be served
```

Comment:
nit: can we just add an example here for the link format too ?

Reply:
Will do